### DISCUSSION TITLE
ExplorerPatcher does not change the taskbar. Right-click on the Taskbar crashes and restarts.

### DISCUSSION THREAD

**Discussion by Chris-de-ze** [valinet/ExplorerPatcher#1754] [Q&A]
Hello there, 

I would very much appreciate your kind help to the following strange problem I have!

My setup: Gigabyte laptop Model G5 MD-51US113SO,   i5-11400H @ 2.70GHz, Windows 11 Pro, Version 22H2

I installed ExplorerPatcher 22621.1992.56.1 (the latest version) for the first time.
The taskbar was changed to Win10.
The  Right-click on the Taskbar worked fine bringing up the popup menu with the ExplorerPatcher's settings.

From the ExplorerPatcher's settings (on the very top of the menu) I changed the traskbar to Win11. 
By doing that, then I was never able to bring back the Win10 taskbar and by right-clicking on the Taskbar , the taskbar crashes  and restarts.  I uninstalled and reinstalled the ExplorerPatcher  more than 5 times, and the same thing happens:  the taskbar remains Win11 (does not change to Win10) and by right-clicking on it , the taskbar crashes  and restarts.  

Please HELP.  What change shall I make to my system (registry etc) to bring back the Win10 taskbar?
THANK YOU in advance

**Comment by Chris-de-ze** [comment]
I managed to bring up the ExplorerPatcher's settings menu by running the following commend from a command prompt (as suggested here by [amokkatgithub] https://github.com/valinet/ExplorerPatcher/issues/1387#issuecomment-1530052029:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

Found out that when the taskbar-style is set on Win11, explorer stops working on a right click. 
When the style is set on win10, there is no problem.

THANK YOU VERY MUCH DEAR [amokkatgithub]

**Comment by dafob200** [comment]
I have this exact same problem, on more than one computer.  Right-click causes explorer to crash and restart.  Changing the style to Windows 10 works normally, but not Windows 11 style.

**Comment by Siderite** [comment]
No matter what I set it to, it crashes three times when switching apps then disables itself. And the main reason for using this is that I want the Windows 10 style. Vali! Heeelp!

**Reply by pyrates999** [reply]
I would uninstall explorer patcher until this is fixed then.

**Reply by Gerboa** [reply]
Is there an ETA for an ExplorerPatcher fix?

**Reply by Chris-de-ze** [reply]
A new version of ExplorerPatcher was released few days ago. The new version works OK with the new Win11 build. 
However, you have to uninstall first the previous version of ExplorerPatcher and then install the new release; otherwise it might not work well.

**Comment by Chris-de-ze** [comment]
I had the same problem after the following windows update:

- 2023-07 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5028254)

I uninstalled the above update from the "Update History" and at the same time I paused updates until 13/Sep. 
Now, explorer patcher works fine.

**Comment by pyrates999** [comment]
In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

**Comment by roadparc** [comment]
Any ETA for when this will be fixed?
I just uninstalled Explorer patcher because of this problem and also the task switcher does not work.

**Reply by Chris-de-ze** [reply]
A new version of ExplorerPatcher was released few days ago. The new version works OK with the new Win11 build.
However, you have to uninstall first the previous version of ExplorerPatcher and then install the new release; otherwise it might not work well.

**Comment by sohmuijai** [comment]
same here, after switching to Win11 taskbar, right-clicking the taskbar crashes it. I had to uninstall and reinstall to switch back to win10 taskbar. and after that, even right-clicking to open Explorer pinned button on the taskbar also crashes it sometimes.

**Reply by Poopooracoocoo** [reply]
State your ExplorerPatcher version and OS build.

**Reply by sohmuijai** [reply]
thanks for replying
but i no longer have local admin on my work laptop and cannot use this anymore..
is it possible to have a current user installer?.....

**Reply by pyrates999** [reply]
Not that I know of as it requires system privileges to run.
